docs/source/certificates/certificates.md

```diff
@@ -0,0 +1,173 @@
+## Certificate Management Guide
+
+**Audience**: Fabric network admins
+
+This guide provides a brief overview, resources, and technical details for Fabric network administrators to manage certificates (certs).
+
+### Key Concepts
+
+**Registration** – A registration is a username (enrollment ID) and password stored in the certificate authority (CA). This is created by a CA admin level user and has no expiration. The registration also contains any roles and attributes required.
```

Comment:
I don't understand why your couple registration, which is a Fabric-CA feature, and certificate management for Fabric core. 

What if I want to use a commercial CA?